ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -94,37 +100,52 @@ Control {
                     ]
                 }
                 ScrollBar.vertical: StatusScrollBar { }
-                delegate: StatusListItem { // TODO custom delegate
-                    width: GridView.view.cellWidth - Style.current.smallPadding
-                    height: GridView.view.cellHeight - Style.current.smallPadding
-                    title: model.name
-                    statusListItemTitle.font.pixelSize: 17
-                    statusListItemTitle.font.bold: true
-                    subTitle: model.description
-                    tertiaryTitle: qsTr("%n member(s)", "", model.membersCount)
-                    asset.name: model.image ?? model.name
-                    asset.isImage: asset.name.startsWith(Constants.dataImagePrefix)
-                    asset.isLetterIdenticon: !model.image
-                    asset.color: model.color
-                    asset.width: 40
-                    asset.height: 40
-                    border.width: 1
-                    border.color: Theme.palette.baseColor2
-                    components: [
-                        StatusIcon {
-                            visible: model.memberRole === Constants.memberRole.owner ||
-                                     model.memberRole === Constants.memberRole.admin ||
-                                     model.memberRole === Constants.memberRole.tokenMaster
-                            anchors.verticalCenter: parent.verticalCenter
-                            icon: "crown"
-                            color: Theme.palette.directColor1
+                delegate: StatusCommunityCard {
```

Comment:
The new delegate drops quite a large shadow that is cropped on the external edges

<img width="285" alt="Screenshot 2024-03-01 at 09 32 52" src="https://github.com/status-im/status-desktop/assets/47811206/a4b9b6cf-9abe-4b5a-b020-17519fb3a756">


One way to fix this would be to enable clipping in the root control and disable clipping in the GridViews. WDYT?

Reply:
I corrected the shadow. other than this it's a clipping thing - need to check further how to adapt this